build.gradle

```diff
@@ -12,6 +12,7 @@
 import java.util.concurrent.Callable
 import org.opensearch.gradle.test.RestIntegTestTask
 import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
+import org.gradle.api.internal.tasks.testing.junit.JUnitTestFramework
```

Comment:
Minor, you probably don't need this import anymore

Reply:
Ah. Missed it. Pushed the change! Thanks